packages/components/src/__future__/Tag/_docs/Tag.stories.tsx

```diff
@@ -1,94 +1,52 @@
 import React from "react"
-import { Meta, StoryFn } from "@storybook/react"
-import { TagIcon } from "~components/Icons"
-import { ComponentDocsTemplate } from "~storybook/components/DocsContainer"
-import { StickerSheet } from "~storybook/components/StickerSheet"
-import { Tag } from ".."
+import { Meta, StoryObj } from "@storybook/react"
+import { AcademyIcon, ActionOffIcon, AddIcon, TagIcon } from "~components/Icons"
+import { Tag, TagColorKeys } from ".."
+import styles from "./TagStories.module.scss"
 
 export default {
-  tags: ["autodocs"],
-  title: "KAIO-staging/Tag",
+  title: "Components/Tag",
   component: Tag,
-  parameters: {
-    docs: {
-      container: ComponentDocsTemplate,
-    },
-    isInKaio: true,
-    installation: [
-      "yarn add @kaizen/components",
-      "import { Tag } from `@kaizen/components/future`",
-    ],
-    resourceLinks: {
-      sourceCode:
-        "https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__future__/Tag",
-      figma:
-        "https://www.figma.com/file/hANEprZKom5Br1IBXhE8mr/%F0%9F%A7%AA-Kaizen-Labs?type=design&node-id=28588-183332&mode=design&t=VHZDCTJUpaon8PXg-0",
-    },
-  },
 } satisfies Meta<typeof Tag>
 
-export const Playground: StoryFn<typeof Tag> = args => (
-  <Tag {...args}>My tag</Tag>
-)
+export const Playground: StoryObj = {
+  render: args => <Tag {...args}>My tag</Tag>,
+}
```

Comment:
We usually like to have the Canvas open for Playground stories, thus the extra params.

```suggestion
export const Playground: Story = {
  parameters: {
    docs: {
      canvas: {
        sourceState: "shown",
      },
    },
  },
}
```